.idea/misc.xml

```diff
@@ -1,5 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="DesignSurface">
```

Comment:
not sure we need this change (or the other .xml files in idea folder). they have some values that are local to your environment

Reply:
idk where these changes came from, thought gitIgnore took care of this, guess not